src/components/DDLText/DDLText.tsx

```diff
@@ -27,10 +27,13 @@ export interface IDDLTextEvent {
   target: { value: string };
 }
 
-const renderHeader = (s: string) => { return (
-  <Col>
-  <h4>{s} : </h4>
-  </Col>); };
+const renderHeader = (s: string): JSX.Element => {
```

Comment:
This is only used with props.header as an argument.
Can be used as a method of the components.